tests/e2e/docker_test.sh

```diff
@@ -1,55 +0,0 @@
-#!/bin/bash
```

Comment:
is there another build and run test script that is preferred for collaborators?

Reply:
Not yet. Tests are embarrassingly sparse on this project. I removed it because it's not actually complete, and didn't want to give the impression that it was testing anything. Let's flag this as a todo for the next minor version.